Client/Frontend/Browser/OpenInHelper.swift

```diff
@@ -104,7 +105,11 @@ class OpenPassBookHelper: NSObject, OpenInHelper {
         }
         let passLibrary = PKPassLibrary()
         if passLibrary.containsPass(pass) {
-            UIApplication.shared.openURL(pass.passURL!)
+            if #available(iOS 10.0, *) {
+                UIApplication.shared.open(pass.passURL!, options: [:], completionHandler: nil)
```

Comment:
No force unwraping please :). you can do a `if passLibrary.containsPass(pass), let passURL = pass.passURL {`